src/xenia/app/emulator_window.cc

```diff
@@ -226,6 +229,36 @@ bool EmulatorWindow::Initialize() {
   }
   main_menu->AddChild(std::move(gpu_menu));
 
+  // User Language menu
+  auto ulang_menu = MenuItem::Create(MenuItem::Type::kPopup, L"&User Language");
+  {
+	  ulang_menu->AddChild(
```

Comment:
It'd be cool if we could ping the game to see what languages it supports before constructing this menu. Dunno if that's possible though.

Reply:
value = 0x00000009;  // Portuguese